### ISSUE TITLE
ERROR -Altay 3.0.7

### ISSUE THREAD

**Issue by ghost** [solo5star/HotAirBalloon#1]
Fatal error: Declaration of solo\hotairballoon\Vehicle::isRiding() must be compatible with pocketmine\entity\Entity::isRiding(): bool in C:\Users\ghkst\Desktop\Altay-master\plugins\HotAirBalloon-master\src\solo\hotairballoon\Vehicle.php on line 177

**Comment by solo5star** [comment]
Oops, I haven't considered Altay api

**Comment by ghost** [comment]
Just only PMMP ?


Not update for Altay ?

**Comment by solo5star** [comment]
Yeah, Only tested in pmmp. but it may work with altay if you rename some functions overlapped with altay's.

**Comment by ghost** [comment]
What should I do? Can you help me?

**Comment by SuperAdam47** [comment]
@solo5star create altay branch
